### ISSUE TITLE
Unknown version 10 of op_mob when given 'mobileToDesktop: true' as option

### ISSUE THREAD

**Issue by wessberg** [browserslist/browserslist#443]
Hey there,

Consider this:

```typescript
const result = browserslist([
    "dead",
]);

// result is:
// [ 'bb 10', 'bb 7', 'ie 10', 'ie 9', 'ie 8', 'ie 7', 'ie 6', 'ie 5.5', 'ie_mob 11', 'ie_mob 10', 'op_mob 12.1', 'op_mob 12', // 'op_mob 11.5', 'op_mob 11.1', 'op_mob 11', 'op_mob 10', 'samsung 4']

const secondResult = browserslist(result);

// secondResult is identical to result:
// [ 'bb 10', 'bb 7', 'ie 10', 'ie 9', 'ie 8', 'ie 7', 'ie 6', 'ie 5.5', 'ie_mob 11', 'ie_mob 10', 'op_mob 12.1', 'op_mob 12', // 'op_mob 11.5', 'op_mob 11.1', 'op_mob 11', 'op_mob 10', 'samsung 4']
```

This is fine and works as expected.

Now, consider the same example, but where the `mobileToDesktop` option is given:

```typescript
Consider this:
const result = browserslist([
    "dead",
]);

// result is:
// [ 'bb 10', 'bb 7', 'ie 10', 'ie 9', 'ie 8', 'ie 7', 'ie 6', 'ie 5.5', 'ie_mob 11', 'ie_mob 10', 'op_mob 12.1', 'op_mob 12', // 'op_mob 11.5', 'op_mob 11.1', 'op_mob 11', 'op_mob 10', 'samsung 4']

// This throws with: 'Unknown version 10 of op_mob'
const secondResult = browserslist(result, {mobileToDesktop: true);
```

This throws with: `Unknown version 10 of op_mob`.

One way a user may run into this is by providing a normalized browserslist that includes `op_mob 10` (such as what you get when you pass `"dead"` as an option) to `@babel/preset-env` which takes that browserslist and then normalizes it, but with the `mobileToDesktop: true` option.

**Comment by ai** [comment]
Thanks for good report.

Fixed 81ace9ca86379e23573faac99afce11fa5e86b9b

**Comment by ai** [comment]
The fix was released at 4.8.6.

If you like how fast we are fixing the problems, feel free to talk to your manager about funding big refactoring of Browserslist to prevent next [ESLint-like malicious issue](https://eslint.org/blog/2018/07/postmortem-for-malicious-package-publishes).

https://opencollective.com/browserslist
